### PR TITLE
Minor rewrite of the content for the English version

### PR DESCRIPTION
**Pull request by mehmandarov**

### CODE REVIEW

**Comment by markwest1972** [comment]
tidsfrist?

**Comment by mehmandarov** [comment]
Ingen tidsfrist, @markwest1972, bare ta en titt når du får tid. Continuous improvement. :-)

File: _data/en/contribute.yml [hunk 1]
```diff
@@ -1,22 +1,22 @@
 name: Contribute
 items:
-  - title: Become an active member of javaBin?
+  - title: Want to become an active member of javaBin?
     content:
-        Want to join actively in javaBin and organize meetups and JavaZone? In Oslo, we meet every Tuesday
-        18:00 PM at <a href="https://www.google.com/maps/place/Pilestredet+56,+0167+Oslo">Teknologihuset</a>,
-        just show up at one of our evenings. In the regions we meet less frequently - show up at a
-        meetup event and talk with us.
+        Want to join javaBin and help us organizing meetups and JavaZone? In Oslo, we meet every Tuesday
```
Comment (gdubya) [inline _data/en/contribute.yml]: Perhaps change "help us organizing" to "help us to organize"?

File: _data/en/contribute.yml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+        18:00 PM at <a href="https://www.google.com/maps/place/Pilestredet+56,+0167+Oslo">Teknologihuset</a>.
```
Comment (gdubya) [inline _data/en/contribute.yml]: "PM" is unnecessary. Either 6 PM or 18:00 ;)

Comment (gdubya) [inline _data/en/contribute.yml]: Ok, how about "In Oslo we meet at 18:00 every Tuesday at [Teknologihuset](https://www.google.com/maps/place/Pilestredet+56,+0167+Oslo); other regions meet less frequently, so [check out your regions upcoming events](http://java.no/#locations) for more details. Just show up at any of the meetings or scheduled meetups and say hi!"

File: _data/en/contribute.yml [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
         you are more than welcome to join us! Please contact us at <a href="mailto:javazone@java.no">javazone@java.no</a>
-        if you have ideas or want to join.
+        if you have any ideas, or want to join the crew.
     imagePath: /img/logos/javazone-logo.jpg
 
   - title: Do you like to code? Contribute on GitHub!
     content:
-        javaBin is run by volunteery developers and because of that we code on a variety of projects - all of which
-        are Open Source and available on <a href="http://github.com/javabin">Github</a>.
+        javaBin is run by volunteer developers. This means that we develop most of our systems ourselves – all of them
```
Comment (gdubya) [inline _data/en/contribute.yml]: Remove "developers"? Testers, project managers, etc. are welcome too ;)

File: _en/0001-01-01-membership.html [hunk 4]
```diff
@@ -10,36 +10,39 @@
 <div class="row">
     <div class="col-md-8 col-md-offset-2 center-justified">
         <p>
-            javaBin is open for everyone who wants to be a member. Being a member with a paid membership means that you support
-            the Java community in Norway and helping us organize meetups and JavaZone.
+            javaBin is open for everyone who wants to become a member. Being a paying member means that you support
+            the Java community in Norway and help us organize various events throughout the year, and JavaZone.
```
Comment (gdubya) [inline _en/0001-01-01-membership.html]: javaBin is open to anyone who wants to become a member. Every membership supports the Java community in Norway and helps us to organize various events throughout the year, including JavaZone.

File: _en/0001-01-01-membership.html [hunk 5]
```diff
@@ -10,36 +10,39 @@
...
         </h3>
 
         <p>
             Membership in javaBin is included in the JavaZone ticket, so if you have bought a ticket to JavaZone or have received this
-            by your employer, then you are already a member of javaBin for one year. If you are not going to JavaZone, you can buy a single membership.
-            Price for this is 500 NOK for 12 months membership. Students with a valid student ID can apply for free membership.
-
-            If you want student membership, please send an email to <a href="mailto:medlemskap@java.no">medlemskap@java.no</a>.
+            from your employer, then you are already a member of javaBin for that year. If you are not going to JavaZone, you can 
+            buy a separate membership. Price for this is 500 NOK for a 12-month membership.
```
Comment (gdubya) [inline _en/0001-01-01-membership.html]: If you are not going to JavaZone then you can buy a separate 12-month membership for 500 NOK.

File: _en/2015-08-11-about.html [hunk 6]
```diff
@@ -14,24 +14,27 @@
             Vi har avdelinger over hele landet – Stavanger, Bergen, Trondheim, Vestfold, Sørlandet og Oslo. Hver eneste måned skjer
             det kule ting i javaBin-regi – både meetups, studentarrangementer, arrangementer for barn, og mye annet spennende!-->
 
-            javaBin is one of the largest community groups in Norway. It is driven by volunteer enthusiasts from across the country.
-            We have branches throughout the country - Stavanger, Bergen, Trondheim, Vestfold, Southern Norway and Oslo. We organize events
-            almost every month in all javaBin branches - both meetups, student events, events for kids, and much more exciting stuff!
+            javaBin is one of the largest communities in Norway. It is driven by volunteer enthusiasts from across the country. 
+            We have branches throughout the country – in Stavanger, Bergen, Trondheim, Vestfold, Southern Norway and Oslo. 
+            We organize events almost every month in all branches of javaBin. It is both meetups, events for students,
```
Comment (gdubya) [inline _en/2015-08-11-about.html]: ... in all branches of javaBin, including meeting, events for students, events for kids, and other exciting stuff!"

File: _en/2015-08-11-about.html [hunk 7]
```diff
@@ -14,24 +14,27 @@
...
+            events for kids, and other exciting stuff!
         </p>
         <p>
             <!-- Hvert år i september er det duket for javaBins store happening – <a href="http://javazone.no">JavaZone</a>.
             Det stemmer! En av verdens største Java-konferanser er drevet av en frivillig organisasjon! 3.000 deltakere,
             200 foredragsholdere, 45 partnerbedrifter og generelt sett god stemning! -->
 
-            That's right! One of the world's biggest Java conferences is run by an non-profit organization! 3,000 participants,
-            200 lecturers, 45 partner companies and general good mood!
+            Our biggest happening of the year – <a href="http://javazone.no">JavaZone</a> – takes place every September.
```
Comment (gdubya) [inline _en/2015-08-11-about.html]: s/happening/event

File: _en/2015-08-11-about.html [hunk 8]
```diff
@@ -14,24 +14,27 @@
...
         </p>
         <p>
             <!--Bli med på et javaBin-arrangement da vel! Og er du ekstra interessert, kan du snakke med oss og bli en av arrangørene.
             Vi er over 50 mennesker som bidrar med å skape kule arrangementer for våre nesten 3.000 medlemmer!-->
 
-            Join a javaBin event! And if you are really interested, you can talk with us and become one of the organizers.
-            We have over 50 people who contribute to create cool events for our nearly 3,000 members!
+            We would like you to join one of our events! Also, if you are interested, you ask us about becoming one of the organizers.
+            We have over 50 people who contribute to the community, creating events for our nearly 3,000 members! And we always looking
+            for new friends!
```
Comment (gdubya) [inline _en/2015-08-11-about.html]: So what are you waiting for? Come and join one of our events! And if you want to give something back then just ask about becoming an organizer, or how you can contribute in other ways. We currently have over 50 people who contribute, creating events for our nearly 3,000 members! We are always looking for new friends.

**Pull request merged**